src/lj_err.c

```diff
@@ -384,11 +385,15 @@ typedef struct UndocumentedDispatcherContext {
   ULONG ScopeIndex;
   ULONG Fill0;
 } UndocumentedDispatcherContext;
+#else
+typedef void* UndocumentedDispatcherContext;
+#endif
 
 /* Another wild guess. */
-extern void __DestructExceptionObject(EXCEPTION_RECORD *rec, int nothrow);
+extern void LJ_CDECL __DestructExceptionObject(EXCEPTION_RECORD *rec,
+					       int nothrow);
```

Comment:
My concern is build environments which pass extra compiler flags to change the default calling convention, though if said environments are unsupported, then fair enough.